target/linux/ath79/dts/qca9558_nec_aterm.dtsi

```diff
@@ -0,0 +1,379 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+#include "qca955x.dtsi"
+
+/ {
+	chosen {
+		/*
+		 * don't specify bootargs property in DeviceTree to
+		 * enable a console with a default baudrate (9600)
+		 * or passed console= parameter from the bootloader
+		 */
+		/delete-property/ bootargs;
+		stdout-path = &uart0;
+	};
+
+	aliases {
+		led-boot = &led_power_green;
+		led-failsafe = &led_power_red;
+		led-running = &led_power_green;
+		led-upgrade = &led_power_green;
+		label-mac-device = &eth0;
+	};
+
+	i2c {
+		compatible = "i2c-gpio";
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&pmx_gpio_i2c_pins>;
+		sda-gpios = <&gpio 20 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
+		scl-gpios = <&gpio 19 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
```

Comment:
ath79 hardware doesn't support open drain mode.

Reply:
Oh, I see. I added `GPIO_OPEN_DRAIN` to resolve the warning messages printed by the kernel, but indeed, wrong binding should be removed.

Reply:
If there are some warnings I suggest keep it. At least it's harmless.